src/platform.config.ts

```diff
@@ -4,11 +4,27 @@ import { PiecesClient } from "pieces-copilot-sdk";
 let BASE_URL: string;
 let WS_URL: string;
 
-// Define your port here 
+// Define your port based on the operating system
+let PORT: number;
+
+// Define your port here
 // LINUX PORT=5323
 // For Windows and MacOs PORT=1000
```

Comment:
We can remove this as these are already mentioned in the docs, or probably you can refer the docs here

Reply:
Done